examples/object_detection/android-jetpack-compose/app/build.gradle

```diff
@@ -0,0 +1,91 @@
+plugins {
+    id 'com.android.application'
+    id 'org.jetbrains.kotlin.android'
+}
+
+android {
+    namespace 'com.example.objectdetection'
+    compileSdk 33
+
+    defaultConfig {
+        applicationId "com.example.objectdetection"
```

Comment:
Let's go ahead and chance the namespace/app ID to `com.google.mediapipe.examples.objectdetection` to match the existing sample

Reply:
done, also changed the packge folders to match the app ID to match existing samples